app/dashboard/templates/shared/menu.html

```diff
@@ -0,0 +1,307 @@
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+      aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  {% if user.is_authenticated %}
+    {% include 'shared/chat_nav.html' %}
+    {% include 'shared/notifications.html' %}
+  {% endif %}
+</div>
+<div class="collapse navbar-collapse order-1" id="navbarSupportedContent">
+  <nav class="navbar-nav mx-auto gc-megamenu">
+    <a class="nav-link {% if is_townsquare %} selected {% endif %} interior" href="/townsquare">{% trans "Home" %}</a>
+    <div class="nav-item dropdown interior gc-megamenu-fw">
+      <a class="nav-link dropdown-toggle" href="#" id="dropdownProducts" role="button"
+        data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        {% trans "Products" %}
+      </a>
+      <div class="dropdown-menu shadow-lg" aria-labelledby="dropdownProducts">
+        <div class="gc-megamenu-content">
+          <div class="row">
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/bount' in request.path or '/explorer' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url "funder_bounties" %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/bounties_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Bounties" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Development resources at your finger tips" %}</p>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'new_bounty' %}">{% trans "Fund Issue" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'explorer' %}">{% trans "Explore Bounties" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'quickstart' %}">{% trans "Funder Quickstart" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'funder_bounties' %}">{% trans "About Bounties" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/hackathon' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'get_hackathons' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/hackathons_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Hackathons" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Build with the coolest web3 projects" %}</p>
```

Comment:
Wording here is a bit awkward - my suggestion: "Build projects with the coolest web3 companies"